### PR TITLE
VET-104 Clinic details in employees

### PR DESCRIPTION
**Pull request by maclipski**
Dodano funkcjonalność wyświetlenia informacji o klinice z poziomu bazy pracowników. Jednocześnie naprawiony został bug z taska VET-110 - przerzucono na Done.

### CODE REVIEW

File: frontend/src/components/ClientDetails.js [hunk 1]
```diff
@@ -11,7 +11,11 @@ const ClientDetails = ({ client }) => {
 
     useEffect(() => {
         const fetchData = async () => {   
-            const response = await fetch('/api/patients')
+            const response = await fetch('/api/patients', {
+                headers: {
+                  'Authorization': `Bearer ${user.token}`
```
Comment (krawczykaghust) [inline frontend/src/components/ClientDetails.js]: Could you put here also content-type as application/json

File: frontend/src/components/EmployeeDetails.js [hunk 2]
```diff
@@ -1,11 +1,14 @@
...
-import { useState } from 'react';
+import { useEffect, useState } from 'react';
 import { Button, Modal } from 'react-bootstrap'
+import ClinicsDetails from '../components/ClinicDetails'
 
 const EmployeeDetails = ({employee}) => {
     const { dispatch } = useEmployeesContext()
     const { user } = useAuthContext()
+    const [clinics, setClinics] = useState('')
+    const [showed, setShowed] = useState('');
```
Comment (krawczykaghust) [inline frontend/src/components/EmployeeDetails.js]: Additional semicolon here

File: frontend/src/components/EmployeeDetails.js [hunk 3]
```diff
@@ -56,6 +59,19 @@ const EmployeeDetails = ({employee}) => {
         setShowModal(true);
     }
         
+    useEffect(() => {
+        const fetchData = async () => {   
+            const response = await fetch('/api/clinics', {
+                headers: {
+                  'Authorization': `Bearer ${user.token}`
```
Comment (krawczykaghust) [inline frontend/src/components/EmployeeDetails.js]: Could you put here also conte-type as applicaton/json

File: frontend/src/components/EmployeeDetails.js [hunk 4]
```diff
@@ -134,6 +150,13 @@ const EmployeeDetails = ({employee}) => {
             <p><strong>Numer konta: </strong>{employee.numer_konta}</p>
             <p><strong>Adres: </strong>{employee.adres}</p>
             <p><strong>ID Kliniki: </strong>{employee.id_kliniki}</p>
+            <p style={{color: "#E5BA73"}} onClick={() => setShowed(showed => !showed)}><strong>Pokaż szczegóły kliniki:</strong></p>
+            {showed ? 
+            <div id="patient-details">
+                {clinics && clinics.filter(p => (p._id === employee.id_kliniki)).map(p => (
```
Comment (krawczykaghust) [inline frontend/src/components/EmployeeDetails.js]: We are dealing with employees, so it should be e?

**Review by krawczykaghust — CHANGES_REQUESTED** [review]
Small logic and style changes

File: frontend/src/components/EmployeeDetails.js (hunk 4 above)
Comment (maclipski) [reply frontend/src/components/EmployeeDetails.js]: We are dealing with clinics here, changed to 'c'

File: frontend/src/components/EmployeeDetails.js (hunk 3 above)
Comment (maclipski) [reply frontend/src/components/EmployeeDetails.js]: Content-type added

File: frontend/src/components/EmployeeDetails.js (hunk 2 above)
Comment (maclipski) [reply frontend/src/components/EmployeeDetails.js]: This file is 50/50 with and without semicolon, no point in fixing it manually. Would u mind doing it automatically due to next sprint?

**Review by krawczykaghust — APPROVED** [review]
Approved

**Pull request merged**